src/styles/animations.less

```diff
@@ -0,0 +1,38 @@
+@keyframes simpleSlideUp {
+  0% {
+    margin-top: 0%;
+    opacity: 1;
+  }
+  100% {
+    margin-top: -50%;
+    opacity: 0;
+  }
+}
+@keyframes simpleSlideDown {
+  0% {
+    margin-top: -50%;
+  }
+  100% {
+    margin-top: 0%;
+    opacity: 1;
+  }
+}
+
+@keyframes modalSlideDown {
+  0% {
```

Comment:
Good catch, @dtaylor113!  I've removed the animations and the rules as they weren't in use.